src/components/frame/SubscribedColoniesList/SubscribedColoniesList.tsx

```diff
@@ -72,7 +64,7 @@ const SubscribedColoniesList = () => {
             );
           })}
       </div>
-      {canInteractWithNetwork && !isMobile && (
```

Comment:
Removed this so that the create a Colony button still appears in the sidebar when wallet is not connected.